src/lib/addons/feature-event-formatter-md.ts

```diff
@@ -279,93 +525,35 @@ export class FeatureEventFormatterMd implements FeatureEventFormatter {
             : ` constraints from ${oldConstraintText} to ${newConstraintText}`;
     }
 
-    generateStrategyRemoveText(event: IEvent): string {
-        const { createdBy, environment, project, preData } = event;
-        const feature = this.generateFeatureLink(event);
-        return `${createdBy} updated *${feature}* in project *${project}* by removing strategy ${preData?.name} in *${environment}*`;
-    }
-
-    generateStrategyAddText(event: IEvent): string {
-        const { createdBy, environment, project, data } = event;
-        const feature = this.generateFeatureLink(event);
-        return `${createdBy} updated *${feature}* in project *${project}* by adding strategy ${data?.name} in *${environment}*`;
-    }
-
-    generateMetadataText(event: IEvent): string {
-        const { createdBy, project } = event;
-        const feature = this.generateFeatureLink(event);
-        return `${createdBy} updated the metadata for ${feature} in project *${project}*`;
-    }
-
-    generateProjectChangeText(event: IEvent): string {
-        const { createdBy, project, featureName } = event;
-        return `${createdBy} moved ${featureName} to ${project}`;
-    }
-
-    generateFeaturePotentiallyStaleOnText(event: IEvent): string {
-        const { project, createdBy } = event;
-        const feature = this.generateFeatureLink(event);
-
-        return `${createdBy} marked feature toggle *${feature}* (in project *${project}*) as *potentially stale*.`;
-    }
-
-    featureLink(event: IEvent): string {
-        const { type, project = '', featureName } = event;
-        if (type === FEATURE_ARCHIVED) {
-            if (project) {
-                return `${this.unleashUrl}/projects/${project}/archive`;
-            }
-            return `${this.unleashUrl}/archive`;
-        }
+    format(event: IEvent): {
+        text: string;
+        url?: string;
+    } {
+        const { createdBy, type } = event;
+        const { action, path } = EVENT_MAP[type] || {
+            action: `triggered *${type}*`,
+        };
 
-        return `${this.unleashUrl}/projects/${project}/features/${featureName}`;
-    }
+        const context = {
+            user: createdBy,
+            event,
+            strategyTitle: this.getStrategyTitle(event),
+            strategyChangeText: this.generateFeatureStrategyChangeText(event),
+            changeRequest: this.generateChangeRequestLink(event),
+            feature: this.generateFeatureLink(event),
+            project: this.generateProjectLink(event),
+        };
 
-    getAction(type: string): string {
-        switch (type) {
-            case FEATURE_CREATED:
-                return 'created';
-            case FEATURE_UPDATED:
-                return 'updated';
-            case FEATURE_VARIANTS_UPDATED:
-                return 'updated variants for';
-            default:
-                return type;
-        }
-    }
+        Mustache.escape = (text) => text;
```

Comment:
[Globally disables Mustache escaping](https://github.com/janl/mustache.js#variables) so we don't need to specify link variables with `{{{link}}}`, which would be unintuitive.